modules/cli-developer-other.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer-cli-commands.adoc
+
+[id='cli-other-developer-commands-{context}']
+= Other developer CLI commands
+
+== ex
```

Comment:
I would skip `oc ex` commands, this are experimental and may change so I'd not document them.